### PR TITLE
NEW: Recipe for Cmder

### PR DESCRIPTION
**Pull request by gabrielcnr**
Conda recipe for Cmder (https://github.com/cmderdev/cmder)

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
Not that I am against it, but this is just a repackaging.

It has some vendored dependencies such as ``conEmu`` and ``git`` 

As I am not an expert of windows I would prefer to @isuruf or @ocefpaf  or someone else to take a look at this

File: recipes/cmder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  skip: true  # [not win]
+
+test:
+  commands:
+    - if not exist %PREFIX%\\cmder\\Cmder.exe exit 1    # [win]
+
+about:
+  home: https://cmder.net/
+  license: MIT
+  license_file: cmder/LICENSE
```
Comment (marcelotrevisani) [inline recipes/cmder/meta.yaml:24]:
```suggestion
  license_file: LICENSE
```

**Comment by gabrielcnr** [comment]
I usually install Cmder on Windows via downloading the official distributable `cmder.zip` bundle from their releases page on GitHub. I based on the recipe for building `git` for Windows which also uses the same kind of distributable package (without compiling it on the conda build):

https://github.com/conda-forge/git-feedstock/blob/master/recipe/meta.yaml#L20
https://github.com/conda-forge/git-feedstock/blob/master/recipe/bld.bat#L1

**Comment by marcelotrevisani** [comment]
Could you add the license of those bundled dependencies? (``git`` and ``conEmu``)

**Comment by gabrielcnr** [comment]
It looks like Cmder is including all the license files for each individual vendored component they ship.

For example, this is how `git` appears inside Cmder installation (notice the LICENSE.txt):

image

**Review by marcelotrevisani — APPROVED** [review]

**Pull request merged**